src/Build.UnitTests/BuildEventArgsSerialization_Tests.cs

```diff
@@ -66,7 +66,7 @@ public void RoundtripProjectStartedEventArgs()
                 properties: new List<DictionaryEntry>() { new DictionaryEntry("Key", "Value") },
                 items: new List<DictionaryEntry>() { new DictionaryEntry("Key", new MyTaskItem() { ItemSpec = "TestItemSpec" }) },
                 parentBuildEventContext: new BuildEventContext(7, 8, 9, 10, 11, 12),
-                globalProperties: null, // we do not serialize GlobalProperties
```

Comment:
@KirillOsenkov Do you recall if this was deliberate for some reason? I couldn't think of one and assumed this was a bug but since it's explicitly mentioned here I thought I'd ask.

Reply:
I'm guessing either to avoid noise or to avoid data bloat? I'm not sure myself.